usagov/usagov-2021
# USAGOV-2007 - Add script to generate cert/key file for GSAAuth

…s for GSAAuth


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2007

## Description


New script for file generation.  

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



Follow the steps in the Jira ticket, which are in the comment titled "Script Usage", and make sure a cert file, and pubkey file are generated for each CF space you try.  Please try at least 2 spaces, and ensure you have 2 appropriately named files for each space you try.

### Change Requirements



### Validation Steps

N/A

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
